### PR TITLE
Adding coverage for environment update functionality

### LINKED ISSUE #1711: Add CLI test for environment update
```
hammer environment update --help

Usage:
    hammer environment update [OPTIONS]

Options:
 --id ID                              
 --location-ids LOCATION_IDS         REPLACE locations with given ids
                                     Comma separated list of values.
 --name NAME                         Environment name
 --new-name NEW_NAME                  
 --organization-ids ORGANIZATION_IDS REPLACE organizations with given ids.
                                     Comma separated list of values.
 -h, --help                          print help
```

### PR DESCRIPTION
**Pull request by oshtaier**
Added tests for `location-ids` and `organization-ids` parameters only as `name` parameter was already covered in the past. As a bonus - added tests for negative scenario
Closes #1711 

```
$ nosetests tests/foreman/cli/test_environment.py -m test_update_negative -s
.....
----------------------------------------------------------------------
Ran 5 tests in 90.886s

OK
$ nosetests tests/foreman/cli/test_environment.py -m test_update_organization -s
.
----------------------------------------------------------------------
Ran 1 test in 39.432s

OK
$ nosetests tests/foreman/cli/test_environment.py -m test_update_location -s
.
----------------------------------------------------------------------
Ran 1 test in 31.577s

OK
```

### CODE REVIEW

File: tests/foreman/cli/test_environment.py [hunk 1]
```diff
@@ -173,3 +173,85 @@ def test_update(self, test_data):
...
+            self.fail(err)
+
+        result = Environment.update({
+            'name': env_name,
+            'location-ids': new_loc["id"]
+        })
+        self.assertEqual(result.return_code, 0)
+
+        result = Environment.info({'name': env_name})
+        self.assertIn(new_loc["name"], result.stdout['locations'])
```
Comment (elyezer) [inline tests/foreman/cli/test_environment.py]: I think the test would be more complete if the first loc is stored and after asserting the new one is in the list, should assert that the first one is not.

File: tests/foreman/cli/test_environment.py [hunk 2]
```diff
@@ -173,3 +173,85 @@ def test_update(self, test_data):
...
+            self.fail(err)
+
+        result = Environment.update({
+            'name': env_name,
+            'organization-ids': new_org["id"]
+        })
+        self.assertEqual(result.return_code, 0)
+
+        result = Environment.info({'name': env_name})
+        self.assertIn(new_org["name"], result.stdout['organizations'])
```
Comment (elyezer) [inline tests/foreman/cli/test_environment.py]: I think the test would be more complete if the first org is stored and after asserting the new one is in the list, should assert that the first one is not.

**Comment by elyezer** [comment]
Thanks for the bonus.

ACK, pending comments.

**Comment by Ichimonji10** [comment]
ACK pending @elyezer's comments. Also, can you please `s/"/'/g` where appropriate?

**Comment by oshtaier** [comment]
Substituted all of " on '.

Added more verification to really check things mentioned by @elyezer

**Comment by oshtaier** [comment]
nosetests tests/foreman/cli/test_environment.py -m test_update_location -s
.
----------------------------------------------------------------------
Ran 1 test in 34.369s

OK

nosetests tests/foreman/cli/test_environment.py -m test_update_organization -s
.
----------------------------------------------------------------------
Ran 1 test in 47.528s

OK

**Comment by Ichimonji10** [comment]
@elyezer's and my comments have been addressed. APT.

**Pull request merged**